packages/components/src/circular-option-picker/style.scss

```diff
@@ -26,6 +26,7 @@ $color-palette-circle-spacing: 12px;
 	vertical-align: top;
 	transform: scale(1);
 	transition: 100ms transform ease;
+	will-change: transform;
```

Comment:
All sources of documentation (for example https://developer.mozilla.org/en-US/docs/Web/CSS/will-change#via_script and ) I read about `will-change` suggest that we add it programmatically and remove it after it's served its purpose, rather than having it enabled by default. 

Have you considered adding it as a one-off effect in this component and then removing it on cleanup?

Reply:
@tyxla You're thinking of adding it when the popover opens ? We can use a translateZ(0) instead of will-change to get the same result !

Reply:
Have we tested adding it only on `hover`/`focus` ? Or what about using the `contain` CSS property instead?

Reply:
I think adding it on `:hover`/`:focus` can be too late since the browser won't be able to do the optimizations before performing the actual element transformation.